docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,294 @@
+
+Scheduling::
+I can schedule backups to run automatically, at a time and frequency that suits my needs.
+
+Data retention::
+I can configure data retention policies to ensure that backups are kept for the desired length of time.
+
+Restore::
+In the event of a failure, I can quickly and easily restore my cloud Kubernetes environment to a previous state, including all configuration, state, and data.
+
+Testing::
+I can test restores in a development environment before performing a full restore, to ensure that my data and applications are fully recoverable.
+
+
+== Iteration: Monitoring and Alerting
+
+As a user of the service I can perform the following actions:
+
+* I can have a monitoring service that will help me keep track of the health and performance of my cloud Kubernetes environment.
+* I can enable alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority.
+* I can see SLO reports so that I can get an overview of how well my cloud Kubernetes environment performs.
+
+As an engineer of the service I can perform the following actions:
+
+* Access operations runbooks so that I can resolve upcoming alerts
+* Access a monitoring dashboard to get an overview of the cloud Kubernetes environment and health
+
+== Iteration: Metrics
+
+As a user of the service I can perform the following actions:
+
+Collection::
+The service will automatically collect and aggregate metrics from my cloud Kubernetes environment and applications, providing a unified view of all metrics.
+
+Visualization::
+Metrics can be visualized in real-time, with interactive charts and graphs, providing insights into the behavior and performance of my cloud Kubernetes environment and the applications.
+
+Alarming::
+I can set up alarms to alert me when specific conditions are met, such as when an application experiences an error.
+
+Custom dashboards::
+I can create custom dashboards to view metrics that are most important to me, and to quickly identify areas of concern.
+
+
+== Iteration: Billing / Service
+
+As a user of the service I can perform the following actions:
+
+SLA::
+I can request a change for the SLA of my cloud Kubernetes environment
+
+Costs::
+I can view and access my running costs of the cloud Kubernetes environment
+
+As an engineer of the service I can perform the following actions:
+
+SLA::
+When a request for changing the SLA of a Kubernetes environment is approved, I can change the SLA.
+
+SLA::
+I can view the SLA of a Kubernetes environment per customer and per cluster
+
+
+== Iteration: Logs
+
+As a user of the service I can perform the following actions:
+
+Cluster Logs::
+I can access the logs of the Kubernetes control plane via a user-friendly interface
+
+Application Logs::
+I can access the logs of my applications via a user-friendly interface
+
+Log retention::
+I can configure the retention of the logs in the cloud provider's logging stack
```

Comment:
```suggestion
I can configure the retention of the logs in the provided logging stack
```

Reply:
Was not added at the end, as we deleted all information about the logs, except for the Cluster Logs.